pact/setDocumentCategory.pact.test.ts

```diff
@@ -20,7 +20,7 @@ pactWith({ consumer: 'manage-recalls-ui', provider: 'manage-recalls-api' }, prov
   describe('change the category of a document', () => {
     test('can successfully change a category', async () => {
       await provider.addInteraction({
-        state: 'a recall and document exist',
+        state: 'a recall and document history exist',
```

Comment:
Sorry, I think this one needs rolling back/ changing to `a recall and uncategorised document exist`

Reply:
done